### PR TITLE
(GH-28) added the current Cake.Recipe

### LINKED ISSUE #28: Update to latest Cake.Recipe
Current is 2.1.0

### PR DESCRIPTION
**Pull request by nils-a**
Also I introduced a "shim" project to create the NuGet package
instead of the nuspec.

fixes #28

### CODE REVIEW

**Review by gep13 — CHANGES_REQUESTED** [review]
Just a couple minor things that I noticed.

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#---------------------------------#
+cache:
+- tools -> recipe.cake
+
+#---------------------------------#
+#  Skip builds for doc changes    #
+#---------------------------------#
+skip_commits:
+  # Regex for matching commit message
+  message: /(doc).*/
```
[the remark this answers is not in the document]
Comment (nils-a) [reply .appveyor.yml:50]: Darn. Now I have to find where I copied that version. We really need more "file-templates" in VSCode. 😄 (Or should stop copying stuff from left to right...)

[the remark this answers is not in the document]
Comment (gep13) [reply .appveyor.yml:50]: Have you seen my CI/CD assets VSCode Extension?

[the remark this answers is not in the document]
Comment (nils-a) [reply .appveyor.yml:50]: No, I have not. Looking at it now.
Also, I have created an issue in cakerecipe-vscode and suggested we implement downloading the default configurations from the Cake.Recipe docs 😉

File: src/Cake.BuildSystems.Module.sln [hunk 2]
```diff
@@ -4,8 +4,7 @@ VisualStudioVersion = 15.0.26228.4
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{CCCD79DB-5207-4116-905E-D3B85F24A9D7}"
 	ProjectSection(SolutionItems) = preProject
-		..\build.cake = ..\build.cake
-		..\cake.config = ..\cake.config
+		..\recipe.cake = ..\recipe.cake
 		..\README.md = ..\README.md
 		..\ReleaseNotes.md = ..\ReleaseNotes.md
```
[the remark this answers is not in the document]
Comment (nils-a) [reply src/Cake.BuildSystems.Module.sln:10]: Yes. I removed that line. Thanks.

**Review by gep13 — APPROVED** [review]
LGTM

**Pull request merged**